static/bidder-params/mediafuse.json

```diff
@@ -5,22 +5,105 @@
 
   "type": "object",
```

Comment:
Replaced with a copy of `appnexus.json`. Please update your documentation at https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/mediafuse.md to reflect this change.

Reply:
What is to be updated on the mediafuse.md file, any specific change?

Reply:
You should include which parameters are supported for your server adapter alias. You may wish to include a reference to the AppNexus adapter or re-list the available parameters in your mediafuse.md file.

Reply:
@SyntaxNode, We want the mediafuse.json file to be aligned to mediafuse.md file. Can you point out the changes that are to be made here?

Reply:
We ask that you provide a bid parameters table in medafuse.md which details the fields you want publishers to provide. This can be the full list of parameters like we have in https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/appnexus.md or a subset like in https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/districtm.md. As far as I'm concerned, you could link to the appnexus.md file for full documentation. The idea is if a publisher wants to use your adapter, they would look at this file first to understand which bid params are required.

Reply:
I'm okay to use the appnexus.md file for MediaFuse adapter.